test/e2e/scheduling/scheduler_predicates.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package scheduling
```

Comment:
This file could be renamed `test/e2e/scheduling/predicates.go` to save some stuttering.